<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(19, 24, 42);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="loginFrame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>1201</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 48, 79);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="usernameLoginLabel">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>280</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="passLoginLabel">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>390</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lgUserLine">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>310</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 2px solid rgb(121, 121, 121);
color: rgb(255, 255, 255);
border-radius: 8px;
padding-left: 15px;
}

QLineEdit:hover{
	border: 2px solid white;
}

QLineEdit:focus{
	border: 2px solid white;
}</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lgPassLine">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>420</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 2px solid rgb(121, 121, 121);
color: rgb(255, 255, 255);
border-radius: 8px;
padding-left: 15px;
}

QLineEdit:hover{
	border: 2px solid white;
}

QLineEdit:focus{
	border: 2px solid white;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>520</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(16, 91, 151, 255), stop:1 rgba(44, 156, 234, 255));
color: white;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="icon">
      <iconset resource="detechRs.qrc">
       <normaloff>:/images/images/ri_login-circle-fill.png</normaloff>:/images/images/ri_login-circle-fill.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>630</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Don't have an account?</string>
     </property>
    </widget>
    <widget class="QLabel" name="QLabel">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>630</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(45, 158, 236);</string>
     </property>
     <property name="text">
      <string>Register Here.</string>
     </property>
    </widget>
    <widget class="QLabel" name="lgDetechLogo">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>80</y>
       <width>311</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="detechRs.qrc">:/images/images/LOGO-and-TEXT - DETECH.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>601</width>
       <height>741</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="detechRs.qrc">:/images/images/login image.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelResult">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>230</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>934</x>
       <y>630</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>registerButton</zorder>
    <zorder>usernameLoginLabel</zorder>
    <zorder>passLoginLabel</zorder>
    <zorder>lgUserLine</zorder>
    <zorder>lgPassLine</zorder>
    <zorder>loginButton</zorder>
    <zorder>label</zorder>
    <zorder>QLabel</zorder>
    <zorder>lgDetechLogo</zorder>
    <zorder>label_3</zorder>
    <zorder>labelResult</zorder>
   </widget>
   <widget class="QLabel" name="detechLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DETECH</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="detechRs.qrc"/>
 </resources>
 <connections/>
</ui>
